sdk/lusid/utilities/api_client_builder.py

```diff
@@ -94,12 +95,13 @@ def __generate_access_token(configuration, okta_response_handler):
         return api_token
 
     @classmethod
-    def build(cls, api_secrets_filename=None, okta_response_handler=None, api_configuration=None, token=None):
+    def build(cls, api_secrets_filename=None, okta_response_handler=None, api_configuration=None, token=None, correlation_id=None):
         """
         :param str api_secrets_filename: The full path to the JSON file containing the API credentials and optional proxy details
         :param typing.callable okta_response_handler: An optional function to handle the Okta response
         :param lusid.utilities.ApiConfiguration api_configuration: A pre-populated ApiConfiguration
         :param str token: The pre-populated access token to use instead of asking Okta for a token
+        :param str correlation_id: Correlation id for all calls made from the returned ApiClient instance
```

Comment:
Might be a good idea to mention that it will be added to the headers of every request.

Reply:
done